infogami/utils/markdown/markdown.py

```diff
@@ -1497,7 +1504,7 @@ def _handleInlineWrapper (self, line) :
                 
                 x = parts[i]
 
-                if isinstance(x, (str, unicode)) :
+                if isinstance(x, basestring) :
```

Comment:
This reverts an explicit fix for a more obscure fix.

Reply:
From PEP8:
> In Python 2, str and unicode have a common base class, basestring, so you can do:
>     if isinstance(obj, basestring):
> Note that in Python 3, unicode and basestring no longer exist (there is only str)

In fact, [__basestring__](https://docs.python.org/2/library/functions.html#basestring) serves no other purpose in Python 2 then as a replacement for __(str, unicode)__

I still stand by this fix as is because in Python 2 __unicode__ has two different meanings and _both_ are used in this file.
1. __unicode__ as a data type that is directly analogous to Python 3's __str__.
2. __unicode()__ as a function that takes the [optional parameters __encoding__ and __errors__](https://docs.python.org/2/library/functions.html#unicode) that Python 3's [__str()__ function](https://docs.python.org/3/library/stdtypes.html#str) will only accept if the first parameter is a bytes-like object.  This is difficult for us to guarantee.

The [__six__ module](https://six.readthedocs.io/#six.string_types) keeps these two meanings distinct with _six.string_types_ and _six.text_type_.  When we drop all support for Python 2, this approach will make the conversion simple :
* basestring --> str
* unicode --> str
* six.string_types --> str
* six.text_type --> str